src/migrations/20230509065107-recreate-username-field.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        'ALTER TABLE api_tokens RENAME COLUMN token_name TO username;',
```

Comment:
is db.runSql or the entire function a transaction? I think these two should run as a transaction together? Also we should probably copy the data from username -> token_name.

Reply:
I'll look into the transaction part, that's a good point. Yup, I can add copying data as well

Reply:
I can't find anything documenting any transaction taking place using runSql.
I found two places in our migrations where we explicitly start transactions in the SQL passed to runSql. In the migrations where we run several SQL statements they are typically run in the same call to runSql.
I can change this to be inside the same runSql call, and I'll see if I can find more info on transactions

Reply:
Let's have a quick catch up about this

Reply:
Ah interesting. Yeah lets talk!